sample/src/main/res/values/styles.xml

```diff
@@ -8,6 +8,7 @@
         <item name="colorAccent">@color/colorAccent</item>
         <item name="colorControlNormal">@color/colorPrimary</item>
         <item name="android:windowBackground">@color/background</item>
+        <item name="floatingSearchViewStyle">@style/CustomFloatingSearchViewTheme</item>
```

Comment:
That change has nothing to do with the PR, right? Can you send another PR explaining that change?